lib/java/com/google/android/material/slider/BaseSlider.java

```diff
@@ -2897,10 +2905,15 @@ public boolean onTouchEvent(@NonNull MotionEvent event) {
       case MotionEvent.ACTION_MOVE:
         if (!thumbIsPressed) {
           // Check if we're trying to scroll vertically instead of dragging this Slider
-          if (isPotentialVerticalScroll(event)
+          if (!isVertical() && isPotentialVerticalScroll(event)
               && abs(eventCoordinate - touchDownX) < scaledTouchSlop) {
             return false;
           }
+          // Check if we're trying to scroll horizontally instead of dragging this Slider
+          if (isVertical() && isPotentialHorizontalScroll(event)
+              && abs(eventCoordinateY - touchDownY) < scaledTouchSlop / 1.2) {
```

Comment:
Can you explain this `1.2`?

Reply:
It is difficult for me to explain it...
If scaledTouchSlop is not divided by 1.2, it is difficult to slide vertical sliders. Try it.

Reply:
Is this also applicable for the vertical scroll or is it specific to this horizontal scroll?